apps/token/src/components/vega-wallet/vega-wallet-not-running.tsx

```diff
@@ -0,0 +1,45 @@
+import './vega-wallet-not-running.scss';
+
+import { useTranslation } from 'react-i18next';
+
+import { FormGroup, Input } from '@vegaprotocol/ui-toolkit';
+
+interface VegaWalletNotRunningProps {
+  url: string;
+  setUrl: (url: string) => void;
+}
+
+// TODO TFE import do we need this?
```

Comment:
?

Reply:
I've deleted this file. If you try to connect and your wallet isn't running it shows a relevant message